packages/dd-trace/src/plugins/util/web.js

```diff
@@ -102,6 +106,24 @@ const web = {
     return tracer.scope().activate(span, fn)
   },
 
+  // catch errors and apply to active span
+  reactivateAndWrapMiddlewareErrors (fn, req) {
```

Comment:
Why is this needed? The scope manager should already be doing that.

Reply:
I think I was using `reactivate` here inappropriately to try to do the work of both binding the  middleware function to the request span's scope, and invoking that middleware function. I've updated this to `tracer.scope().bind(fn, <request span>).apply(this, arguments)` and updated the method name to `bindAndWrapMiddlewareErrors` to more appropriately reflect the work being done here. I'm not 100% sure i follow where the ScopeManager would do that automatically, but if it's the case that I'm missing something here with how ScopeManager works and this call to `scope().bind` is unnecessary, please let me know